src/MovingHarmonicPotential.cc

```diff
@@ -58,15 +53,17 @@ namespace detail
 /*!
  * \param m Python module to export to
  */
-void export_ImplicitEvaporator(pybind11::module& m)
+void export_MovingHarmonicPotential(pybind11::module& m)
     {
     namespace py = pybind11;
-    py::class_<ImplicitEvaporator, std::shared_ptr<ImplicitEvaporator>>(m,
-                                                                        "ImplicitEvaporator",
-                                                                        py::base<ForceCompute>())
+    py::class_<MovingHarmonicPotential, std::shared_ptr<MovingHarmonicPotential>>(m,
+                                                                                  "MovingHarmonicPotential",
+                                                                                  py::base<ForceCompute>())
```

Comment:
We recently discovered that this way of describing the inheritance of the class leads to very strange (incorrect) behavior in Python, due to some change in pybind11. I think it should be like this instead:
```suggestion
    py::class_<HarmonicBarrier, std::shared_ptr<HarmonicBarrier>, ForceCompute>(m,
                                                                                  "HarmonicBarrier")
```
You will want to propagate this everywhere.